### PR TITLE
Feature/filter 2

### PR DESCRIPTION
**Pull request by richardverheyen**
Updated version of the people filter, allows users to click on links in the footer which return the people page with an automatically applied filter. Once the page has been loaded, the stored filter in sessionStorage is immediately deleted, and the page runs as normal.

Commenting and labelling of all the jQuery in main.js isn't perfect, but I'll do that as you're reading over this, so expect another commit soon.

### CODE REVIEW

File: src/templates/partials/footer.hbs [hunk 1]
```diff
@@ -29,3 +33,7 @@
     </div>
   </section>
 </footer>
+</ul>
+</div>
+</section>
+</footer>
```
Comment (janwerkhoven) [inline src/templates/partials/footer.hbs]: Something is wrong here, we have 2 `</footer>`

File: src/styles/about/_people.scss [hunk 2]
```diff
@@ -83,7 +83,7 @@
           height: 0;
           transition: all ease-out 400ms;
           &.active {
-            height: 200px;
+            height: 240px;
```
Comment (janwerkhoven) [inline src/styles/about/_people.scss:86]: Because we added 1 line to the dropdown?

File: src/js/main.js.orig [hunk 3]
```diff
@@ -1,238 +0,0 @@
...
-  $('.tiles.people li').on('click', function() {
-    var $bio = $(this).find('.bio').clone();
-    var $img = $(this).find('.img').clone();
-    var $modal = $('#modals .modal');
-    $('#modals .modal .content').html('').append($img).append($bio);
-<<<<<<< HEAD
-    openModal();
-=======
-    closeModal();
->>>>>>> b825156c9b33dae118f2d4552f05c2762608795e
```
Comment (janwerkhoven) [inline src/js/main.js.orig:203]: Why is there a `.orig` file in our staging environment?

File: src/js/main.js [hunk 4]
```diff
@@ -158,10 +156,29 @@ $(document).ready(function() {
 
   //Display the active filter
   $('#filter ul button').on('click', function(event) {
+    $element = $(this);
+    if ($element.hasClass('selected')) {
+      console.log('Jan surprise!');
```
Comment (janwerkhoven) [inline src/js/main.js]: Hmm let's not leave `console.log()`s in public space. These are only for debugging, not for production. Please remove.

File: src/js/main.js [hunk 5]
```diff
@@ -186,8 +202,21 @@ $(document).ready(function() {
...
+  $('footer a.filter').on('click', function() {
+    var $filter = $(this).attr('class').replace('filter ', '');
+    sessionStorage.setItem('storedFilter', $filter);
   });
 
+  //on page load, check if there is a stored filter from a page nagivation
+  if (sessionStorage.getItem('storedFilter')) {
+    applyFilters();
+  } else {
+    console.log('No stored filters');
```
Comment (janwerkhoven) [inline src/js/main.js]: Doesn't belong in production. For development only. Please remove.

File: src/js/main.js [hunk 6]
```diff
@@ -186,8 +202,21 @@ $(document).ready(function() {
 
     // Prevent the parent dropdown from closing
     event.stopPropagation();
+  }
+
+  // Pre-apply filters when user navigates to People using the footer links
+  $('footer a.filter').on('click', function() {
+    var $filter = $(this).attr('class').replace('filter ', '');
+    sessionStorage.setItem('storedFilter', $filter);
```
Comment (janwerkhoven) [inline src/js/main.js:207]: Good stuff mate. I like how you also first declare `$filter`. 🎉

File: src/templates/partials/footer.hbs (hunk 1 above)
Comment (janwerkhoven) [reply src/templates/partials/footer.hbs]: Can you remove these 4 lines?

**Review by janwerkhoven — CHANGES_REQUESTED** [review]
The footer may break because 4 lines have been added. And ideally also remove all console logs and debuggers before going to production.

File: src/styles/about/_people.scss (hunk 2 above)
Comment (richardverheyen) [reply src/styles/about/_people.scss:86]: That's right

File: src/templates/partials/footer.hbs (hunk 1 above)
Comment (richardverheyen) [reply src/templates/partials/footer.hbs]: Yeah there was some sort of error with the grunt, I ran it again and the extras disappeared

File: src/js/main.js.orig (hunk 3 above)
Comment (richardverheyen) [reply src/js/main.js.orig:203]: I forgot to go `git clean -f`

**Review by janwerkhoven — APPROVED** [review]

**Comment by janwerkhoven** [comment]
⛵️

**Pull request merged**